robottelo/host_helpers/capsule_mixins.py

```diff
@@ -199,26 +199,46 @@ def cutoff_host_setup_log(self, proxy_hostname, hostname):
             f'sshpass -p "{settings.server.ssh_password}" scp -o StrictHostKeyChecking=no root@{proxy_hostname}:{log_path} {old_log}'
         )
         # make sure the system can't communicate with the git directly, without proxy
-        assert (
-            self.execute(
-                f'firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 1 -d $(dig +short A {hostname}) -j REJECT && firewall-cmd --reload'
-            ).status
-            == 0
-        )
+        if settings.server.is_ipv6:
```

Comment:
I suggest always creating both rules (for ipv4 and ipv6) regardless of `is_ipv6`. DNS should always return all records, even on ipv4-only or ipv6-only machines.
Even a `is_ipv6==True` machine can have some ipv4 connectivity and bypass the limitations imposed by the test/firewall.
You can of course check whether a record exists and if it doesn't, then don't create the rule. But if it doesn't exist, it is a problem by itself.